bin/setup.dart

```diff
@@ -63,7 +63,8 @@ void main() async {
     'cmake',
     [
       '-S',
-      webcryptoRoot.resolve('src').toFilePath(),
+      Directory(webcryptoRoot.toFilePath() + Platform.pathSeparator + 'src')
+          .path,
```

Comment:
Should we do something like:
```dart
/// Join paths without duplicate path separators.
String joinPaths(String prefix, String suffix) {
  if (!prefix.endsWith(Platform.pathSeparator)) {
    prefix += Platform.pathSeparator;
  }
  return prefix + suffix;
}
```

Reply:
In the systems I'm aware of, directory1//directory2 == directory1/directory2, so it should not be necessary. But it would not hurt ofc if you feel this is cleaner.

Reply:
It would be nice to be clean. Mostly because it might affect other things down the line (probably won't).

Reply:
I added it in https://github.com/google/webcrypto.dart/pull/40/commits/4754ccc23fac6470f58207e16f0e5da9df07c6c3 (your approach worked perfectly in my test).